### PR TITLE
re-add FEATURE_SHOTTIMER Flag

### PR DESCRIPTION
**Pull request by LoQue90**

### CODE REVIEW

File: src/display/displayCommon.h [hunk 1]
```diff
@@ -238,7 +238,7 @@ void displayLogo(String displaymessagetext, String displaymessagetext2) {
  * @brief display shot timer
  */
 void displayShottimer(void) {
-    if ((machineState == kBrew || brewSwitchState == kBrewSwitchFlushOff) && SHOTTIMER_TYPE == 1) {
+    if ((machineState == kBrew || brewSwitchState == kBrewSwitchFlushOff) && FEATURE_SHOTTIMER == 1 && SHOTTIMER_TYPE == 1) {
```
Comment (kjyv) [inline src/display/displayCommon.h:241]: I think this got broken during your rebase, theree was supposed to be a 
`if (FEATURE_SHOTTIMER == 0) {
    return;
}` at the top of displayShottimer, then these can be removed

Comment (LoQue90) [reply src/display/displayCommon.h]: urgs, this was my fault, I´m sorry!
Fixed and rebased again, ready for merge.

File: src/display/displayCommon.h [hunk 2]
```diff
@@ -250,7 +255,7 @@ void displayShottimer(void) {
 
         displayBrewtime(48, 25, timeBrewed);
 
-        displayWaterIcon(119, 1);
+        displayWaterIcon(119, 1);
```
Comment (fiendie) [inline src/display/displayCommon.h:258]: Trailing whitespace

Comment (LoQue90) [reply src/display/displayCommon.h]: done

File: src/display/displayCommon.h [hunk 3]
```diff
@@ -238,6 +238,11 @@ void displayLogo(String displaymessagetext, String displaymessagetext2) {
  * @brief display shot timer
  */
 void displayShottimer(void) {
+
```
Comment (kjyv) [inline src/display/displayCommon.h:241]:
```suggestion
```

**Review by kjyv — APPROVED** [review]

**Pull request merged**